src/index.jsx

```diff
@@ -25,13 +26,17 @@ const { ADMIN_ROLE, USER_ROLE } = AUTH_ROLES;
 // specify component to render if access allowed, redirectPath if access denied
 // use roles from AUTH_ROLES; add to roles list
 
+// useNavigate for redirects
+// const navigate = useNavigate();
+
 ReactDOM.render(
   <React.StrictMode>
     <CookiesProvider>
       <Router>
+        <NavigationBar />
```

Comment:
Please remove nav bar from `index.jsx`

Reply:
Is there a reason we don't have it on `index.jsx` instead of having to manually add it per page? I feel like having navbar/footer being inserted manually on each page introduces an unnecessary moving part to the project

Reply:
I guess you're right yeah. However, I think that change should be made in another PR (where the navbar is simultaneously removed from all other pages).

I want to limit the scope of this PR so there isn't too much changed at once bc if there's an issue it might be hard to track changes.